hashes.lst

```diff
@@ -27,19 +27,20 @@
 # and the first match wins.  Therefore, the empty prefix (traditional
 # DES) must be last.
 #
-#name    gs_suffix  h_prefix  nrbytes  flags
-yescrypt :          $y$       16       STRONG,ALT,DEFAULT
-scrypt   :          $7$       16       STRONG
-bcrypt   _a         $2a$      16       STRONG,ALT,FREEBSD,NETBSD,OPENBSD,OWL,SOLARIS,SUSE
-bcrypt   _b         $2b$      16       STRONG,ALT,FREEBSD,NETBSD,OPENBSD,OWL,SOLARIS,SUSE
-bcrypt   _x         $2x$      16       STRONG,ALT,FREEBSD,NETBSD,OPENBSD,OWL,SOLARIS,SUSE
-bcrypt   _y         $2y$      16       STRONG,ALT,FREEBSD,NETBSD,OPENBSD,OWL,SOLARIS,SUSE
-sha512   :          $6$       15       STRONG,GLIBC,FREEBSD,SOLARIS
-sha256   :          $5$       15       STRONG,GLIBC,FREEBSD,SOLARIS
-md5      :          $1$       9        GLIBC,FREEBSD,NETBSD,OPENBSD,SOLARIS
-sunmd5   :          $md5      8        SOLARIS
-sha1     :          $sha1     20       NETBSD
-nthash   :          $3$       7        FREEBSD
-des_xbsd :          _         3        FREEBSD,NETBSD,OPENBSD,OSX
-des_big  :          :         2        :
-des      :          :         2        GLIBC,FREEBSD,NETBSD,OPENBSD,SOLARIS,OSX
+#name         gs_suffix  h_prefix  nrbytes  flags
+yescrypt      :          $y$       16       STRONG,ALT,DEFAULT
+gost-yescrypt :          $gy$      16       STRONG,ALT
```

Comment:
If you use `gost_yescrypt` for the name instead of `gost-yescrypt`, then you won't have to change gen-hashes.awk.  This name isn't used anywhere but the `--enable-hashes` configure argument and the `INCLUDE_` macros (where it has to be an underscore) so I don't see any strong reason to insist on using a dash (maybe we should make configure automatically and universally map dashes to underscores in that argument).

We should make all of these names consistent with John the Ripper, as @solardiz pointed out wrt the crypt.conf project, but that should be a separate patch.

Reply:
The next rebased push will fix this.